packages/react-native/scripts/ios-prebuild/cli.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ * @oncall react_native
+ */
+
+const yargs = require('yargs');
+
+/*::
+ */
+
+// CI can't use commas in cache keys, so 'macOS,variant=Mac Catalyst' was creating troubles
+// This map that converts from platforms to valid Xcodebuild destinations.
+const platformToDestination /*: $ReadOnly<{|[Platform]: Destination|}> */ = {
+  ios: 'iOS',
```

Comment:
we only need to build for ios, ios-simulator, mac catalyst. All the others are OOT platforms that have their own fork, so we can't really build for them.